core/modules/double_buffer_flagger.cc

```diff
@@ -0,0 +1,85 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright(c) 2021 Open Networking Foundation
+ */
+
```

Comment:
We can move the entire functionality of this module into QoSMeasure itself, right? We can provide an initialization knob to say if its the leader. Leader would do the metadata update signalling the follower instances.